plugins/base/src/main/kotlin/signatures/JvmSignatureUtils.kt

```diff
@@ -93,20 +92,29 @@ interface JvmSignatureUtils {
     ): Unit = when (a) {
         is AnnotationValue -> toSignatureString(a.annotation, renderAtStrategy, listBrackets, classExtension)
         is ArrayValue -> {
-            text(listBrackets.first.toString())
-            a.value.forEachIndexed { i, it ->
-                group(styles = setOf(TextStyle.BreakableAfter)) {
-                    valueToSignature(it, renderAtStrategy, listBrackets, classExtension)
-                    if (i != a.value.size - 1) text(", ")
-                }
-            }
-            text(listBrackets.second.toString())
+            listParams(a.value, listBrackets) { valueToSignature(it, renderAtStrategy, listBrackets, classExtension) }
         }
         is EnumValue -> link(a.enumName, a.enumDri)
         is ClassValue -> link(a.className + classExtension, a.classDRI)
+        is StringValue -> group(styles = setOf(TextStyle.Breakable)) { text( "\"${a.text()}\"") }
         is LiteralValue -> group(styles = setOf(TextStyle.Breakable)) { text(a.text()) }
     }
 
+    private fun<T> PageContentBuilder.DocumentableContentBuilder.listParams(
+        params: Collection<T>,
+        listBrackets: Pair<Char, Char>?,
+        outFn: PageContentBuilder.DocumentableContentBuilder.(T) -> Unit
+    ) {
+        listBrackets?.let{ text(it.first.toString()) }
+        params.forEachIndexed { i, it ->
+            group(styles = setOf(TextStyle.BreakableAfter)) {
+                this.outFn(it)
+                if (i != params.size - 1) text(", ")
```

Comment:
you can use `lastIndex`

Reply:
It is only for List or Array, not Collection.